pontoon/base/models.py

```diff
@@ -63,7 +63,6 @@
 
 UNUSABLE_SEARCH_CHAR = "☠"
 
-
```

Comment:
Nit: you should restore this line. We keep two blank lines between imports and initializations, and functions/classes.

Reply:
Restored.